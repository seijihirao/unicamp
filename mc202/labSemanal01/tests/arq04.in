10
-997200.762899 
716349.919107 -718086.066268 
-891680.913175 -640132.378481 477814.947153 
633755.493106 -475995.995869 813994.425407 531682.453898 
-254165.815610 57119.494412 300396.127104 196585.974149 -175669.442985 
-272699.788895 351510.806732 626671.040117 -865099.059488 -394506.603717 -47488.158046 
522119.222168 537170.719003 974348.264434 -883604.761811 -393058.682949 480863.168653 -396450.120468 
134272.422037 267448.733863 -502725.371119 302083.773168 471102.075179 -485088.823923 696448.940226 133473.577722 
720064.915795 751579.117241 218069.026190 641103.779207 748310.100007 -286953.313043 -55191.092279 -473213.435582 -850565.579829 
-445965.691191 338038.243249 -955450.515335 817575.895262 -68505.665881 -803782.766899 -837015.382332 753265.711731 -793780.216305 -326805.883415 
