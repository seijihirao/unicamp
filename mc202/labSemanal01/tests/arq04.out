-1.709275644447 
1.232041595834 -1.230174046906 
-1.528150360880 -1.096366200871 0.822595273186 
1.090267902357 -0.814625382228 1.399648817970 0.915059084116 
-0.433852907587 0.100469688545 0.518055092589 0.339864511004 -0.299113540654 
-0.465666553794 0.605793663733 1.078107407816 -1.482522492764 -0.674748484186 -0.079089804425 
0.898643755025 0.924479715173 1.674896884005 -1.514287611741 -0.672263121855 0.827827558723 -0.678084539190 
0.232903026260 0.461500747988 -0.860506471752 0.520951944439 0.811072599083 -0.830233261353 1.197881478543 0.231531806500 
1.238418403898 1.292512668515 0.376740263793 1.102881267589 1.286901386135 -0.490132790335 -0.092311924727 -0.809849105326 -1.457575711364 
-0.763078232904 0.582667949589 -1.637611161899 1.405796426640 -0.115166448549 -1.377272808354 -1.434316738096 1.295407715333 -1.360103386511 -0.558539901683 

-1412.071299527273 582579.348646476865 
